transport/src/test/java/io/netty/channel/PendingWriteQueueTest.java

```diff
@@ -248,9 +248,10 @@ public void operationComplete(ChannelFuture future) throws Exception {
     @Test
     public void testCloseChannelOnCreation() {
         EmbeddedChannel channel = new EmbeddedChannel(new ChannelInboundHandlerAdapter());
+        ChannelHandlerContext context = channel.pipeline().firstContext();
```

Comment:
Close will clear the pipeline if implemented correctly so we need to do this change